tests/ssg_test_suite/common.py

```diff
@@ -308,3 +325,34 @@ def iterate_over_rules():
                 directory=tests_dir, id=full_rule_id, short_id=short_rule_id,
                 files=scripts)
             yield result
+
+
+INSTALL_COMMANDS = dict(
+        fedora=("dnf", "install", "-y"),
+        rhel7=("yum", "install", "-y"),
+        rhel8=("yum", "install", "-y"),
+)
+
+
+def install_packages(domain_ip, platform, packages):
+    machine = "{0}@{1}".format(REMOTE_USER, domain_ip)
+    log_file_name = os.path.join(LogHelper.LOG_DIR, "env-preparation.log")
+
+    with open(log_file_name, 'a') as log_file:
+        print("Installing packages", file=log_file)
```

Comment:
There is something weird about this. When I open `env-preparation.log` in the run log directory, I can see the output of yum command there, and **after** that there is "Installing packages" and the ssh command arguments. It should be before the yum output. It might be something with locking or flushing the log file.

Reply:
Yes, it was the flush.